### ISSUE TITLE
Feature request: timing tests

### ISSUE THREAD

**Issue by DexterMagnific** [libcheck/check#200]
Dear developer,

I know from the doc that timeouts can be set for unit tests.
Is there a way to know how much time a unit test took, regardless of any set timeout ?

Thanks

**Comment by brarcher** [comment]
The XML logging list test duration. Docs [here](https://libcheck.github.io/check/doc/check_html/check_4.html#XML-Logging) and the code creating the duration output [here](https://github.com/libcheck/check/blob/master/src/check_log.c#L307). 

Will that suffice for what you are looking for?

**Comment by DexterMagnific** [comment]
Yes, I was using the stdout output so the duration was not printed. I willo try to make a JSON file out of the statistics. Do you know a JSON structure for storing test results ?

**Comment by brarcher** [comment]
> Do you know a JSON structure for storing test results ?

Not specifically. You may have to make one up. (:
